Project2/style.css

```
@@ -0,0 +1,115 @@
+body {
+    box-sizing: border-box;
+    height: auto;
+    width: 640px;
+}
+
+.mainimage {
+    background-image: url(Images/TheFavouriteWebCardImage.jpg);
+    background-size: cover;
+    height: 380px;
+    position: relative;
+    width: 640px;
+    z-index: 3;
+
+
+}
+
+
+
+.content {
+    background: linear-gradient(to left, #42513F, transparent 190%);
+    height: 100%;
+    width: 640px;
+}
+
+
+.details {
+    left: 410px;
+    padding: 10px;
+    position: relative;
+    top: 1px;
+    width: 200px;
+
+
+}
+
+
+
+
+h1 {
+    bottom: -20px;
+    color: #DB5756;
+    font-size: 20pt;
+    position: relative;
+}
+
+h2 {
+    bottom: 15px;
+    color: #DB5756;
+    font-size: 9pt;
+    position: relative;
+    font-weight: 100;
+    left: 165px;
+    line-height: 8px;
+    word-spacing: 2px;
+}
+
+h3 {
+    bottom: 9px;
+    color: #DB5756;
+    font-size: 11pt;
+    font-weight: 20;
+    line-height: 2px;
+    position: relative;
+    word-spacing: 2px;
+
+}
+
+h4 {
+    bottom: 52px;
+    color: #DB5756;
+    font-size: 9pt;
+    left: 77px;
+    position: relative;
+}
+
+p {
+    color: #FFEEEE;
+    font-size: 8pt;
+    font-weight: 100;
+    left: 70px;
+}
+
+
+.playbutton {
+    bottom: 300px;
+    color: #DB5756;
+    font-size: 90px;
+    left: 225px;
+    opacity: .8;
+    position: relative;
+    z-index: 5;
+}
+
+.bookmarkbutton {
+    bottom: 370px;
+    color: #DB5756;
+    font-size: 20px;
+    left: 874px;
+    position: relative;
+}
+
+.starrating {
+    bottom: 160px;
+    color: #FFEEA6;
+    font-size: 10px;
+    left: 825px;
+    position: relative;
+}
+
+.icons {
+    position: relative;
+    right: 270px;
+    z-index: 10;
```

Is there a reason for the `z-index` of the icons to be _higher_ than that of the play button?
```suggestion
    z-index: 5;
```